gradle.properties

```diff
@@ -20,3 +20,10 @@ org.gradle.parallel=true
 
 #Allows generation of idea/eclipse metadata for a specific subproject and its upstream project dependencies
 ide.recursive=true
+
+
+#---------------------------------
+# Versions
+#---------------------------------
+hadoopVersion=2.6.1
+hiveVersion=0.13.1
```

Comment:
We do! as part of plugins! For now, this change is just pulling out these versions to an external file so that we can easily change these and also maintain a consistent version across all modules.